host/tests/unit/realm-test.ts

```diff
@@ -68,6 +68,105 @@ module('Unit | realm', function (hooks) {
     assert.ok(json.data.meta.lastModified, 'lastModified is populated');
   });
 
+  skip('realm can serve GET card requests with linksTo relationships', async function (assert) {
```

Comment:
this test is failing in the exact way we want it to now--the server response includes everything but the `included` array.